doc/contributor/workarounds.md

```diff
@@ -1,37 +1,36 @@
 # Patching system
 
-The patching system is loaded in `post-boot.rb` and is able to patch stdlib and installed gems. 
-The code implementing the patching itself can be found in `lib/truffle/truffle/patching.rb`.
-It can be disabled by passing the `-Xpatching=false` option.
-
-The patching system works for gems as follows. 
-When a gem *g* is activated, 
-there is a directory `g` in `lib/patches`, 
-and the directory is listed in `lib/truffle/truffle/patching.rb`,  
-the directory is inserted into `$LOAD_PATH` before the original load-paths of the *g* gem. 
-As a result the patching files are loaded first before the original files in the gem.
-The patching file is responsible for loading the original file (if desirable),
-which can be done with `Truffle::Patching.require_original __FILE__`.
+The patching system is loaded in `post-boot.rb` and is able to patch stdlib and
+installed gems.  The code implementing the patching itself can be found in
+`lib/truffle/truffle/patching.rb`. It can be disabled by passing the
+`-Xpatching=false` option.
+
+The patching system works for gems as follows.  When a gem *g* is activated,
+there is a directory `g` in `lib/patches`,  and the directory is listed in
+`lib/truffle/truffle/patching.rb`,   the directory is inserted into `$LOAD_PATH`
```

Comment:
There's an extra space before "the directory."

Reply:
Fixing.